2020-02-Spring/0_arrays_matrix/README.md

```diff
@@ -70,21 +70,33 @@ Go to [Solution](#s2)   [Top](#top)
 <!-- Don't remove -->
 <a name="p3"/>
 
-### 3. PROBLEM 3 TODO :bug:
+### 3. Word Search
 
-Source: TODO :bug:
+Source: [LeetCode](https://leetcode.com/problems/word-search/)
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Given a 2D board and a word, find if the word exists in the grid.
+
+The word can be constructed from letters of sequentially adjacent cell, where "adjacent" cells are those horizontally or vertically neighboring. The same letter cell may not be used more than once.
 
 #### Example Input
 
-If the problem is simple enough, remove this section. TODO :bug:
+board =
+[
+  ['A','B','C','E'],
+  ['S','F','C','S'],
+  ['A','D','E','E']
+]
+
+Given word = "ABCCED", return true.
+Given word = "SEE", return true.
+Given word = "ABCB", return false.
 
 #### Function Signature
 
-TODO :bug:
+* Python: `def exist(board, word)`
+* Java:   `public boolean exist(char[][] board, String word)`
```

Comment:
Recommend doing code blocks instead of inline code specifically because you get syntax highlighting. Feel free to ignore.
````suggestion
Python: 

```python
def exist(board, word):
    # your code here
```

Java:   
```java
public boolean exist(char[][] board, String word) {
    // your code here
}
```
````